### PR TITLE
Issue #701: Improve messaging for confirmation dialogues

### PR DESCRIPTION
**Pull request by katelynienaber**
## Proposed changes

Improved messaging for opening binary files & deleting nodes, according to [issue #701](https://github.com/zowe/vscode-extension-for-zowe/issues/701)

## Release Notes

Milestone: Backlog
Changelog: Improved messaging for opening binary files & deleting nodes

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [x] `npm run vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [x] There is coverage for the code that I have added
- [x] I have tested it manually and there are no regressions found
- [x] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

**Comment by katelynienaber** [comment]
The only code I changed was the localized text, and the localized text in the tests. Don't know why codecov is mad...

**Comment by jellypuno** [comment]
@katelynienaber You added docs and integration test for this PR?

**Comment by katelynienaber** [comment]
> @katelynienaber You added docs and integration test for this PR?

@jellypuno no, its just easier to check all the boxes than exclude integration tests and documentation

**Comment by jellypuno** [comment]
> > @katelynienaber You added docs and integration test for this PR?
> 
> @jellypuno no, its just easier to check all the boxes than exclude integration tests and documentation

@katelynienaber Can you please correct it so I won;t look for it?

**Comment by katelynienaber** [comment]
@zFernand0 The messages have been updated with @crawr's better format!

File: i18n/sample/src/uss/ZoweUSSNode.i18n.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "getChildren.error.response": "Retrieving response from ",
     "getChildren.responses.error.response": "The response from Zowe CLI was not successful",
     "getChildren.responses.open": "Open",
-    "deleteUSSNode.quickPickOption": "Are you sure you want to delete ",
+    "deleteUSSNode.quickPickOption": "Delete {0}? This will permanently remove it from your system.",
     "deleteUSSNode.showQuickPick.yes": "Yes",
```
Comment (crawr) [inline i18n/sample/src/uss/ZoweUSSNode.i18n.json:8]: This should be Delete and Cancel as well to be consistent with the other choices

**Comment by zFernand0** [comment]
Waiting on #851 to be merged

**Pull request merged**